Decred Wallet/Features/Send/ConfirmToSendFundViewController.swift

```diff
@@ -51,14 +55,17 @@ class ConfirmToSendFundViewController: UIViewController, UITextFieldDelegate {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        self.sendAmountLabel.text = "Sending \(self.sendAmount!)"
-        self.feeLabel.text = "with a fee of \(self.fee!)"
-        self.addressLabel.text = "to " + self.destinationAddress!
+        self.sendAmountLabel.text = "\(LocalizedStrings.sending) \(self.sendAmount!)"
+        self.feeLabel.text = "\(LocalizedStrings.withFee) \(self.fee!)"
+        self.addressLabel.text = LocalizedStrings.to + self.destinationAddress!
+        self.passwordTextField.placeholder = LocalizedStrings.passwordHint
+        self.sendBtn.setTitle(LocalizedStrings.send.uppercased(), for: .normal)
+        self.cancelBtn.setTitle(LocalizedStrings.cancel.uppercased(), for: .normal)
```

Comment:
The send and cancel button texts are "Send" and "Cancel", the buttons can use the "send" and "cancel" keys in IB. No need to create IBOutlets for them just to assign text to the buttons.